odoo/addons/base/models/assetsbundle.py

```diff
@@ -201,15 +202,14 @@ def clean_attachments(self, extension):
         """
```

Comment:
The keep_url could be renammed but the idea is that, when cleaning the attachements, we want to remove all attachement except the one we just generated. Since we are in a context were we just generated the attachement, we have the exact url and the corresponding extra. This avoid to duplicate this logic in clean_attachement.

 `clean_attachments(self, 'js', /web/assets/123456/-/web.asset_backend.js, '-'):`
 
 The meaning is, remove all js assets with extra '-' for web.asset_backend except the one with unique 123456